src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    BLACK,
+    WHITE
+}
```

Comment:
It's a good practice to add a semicolon after the last enum constant even though it's not strictly necessary unless you are adding more code to the enum.